token/core/common/validator.go

```diff
@@ -23,6 +27,7 @@ type Context[P driver.PublicParameters, T any, TA driver.TransferAction, IA driv
 	IssueAction       IA
 	Ledger            driver.Ledger
 	MetadataCounter   map[string]int
+	Attributes        map[string][]byte
```

Comment:
If we really need to pass a map and not a struct with a stricter structure, I would suggest at least we pass a `map[AttributeType][]byte` where `AttributeType` is an enum, so that we know what to expect.

Reply:
👍